src/test/java/BogdanSandu/tests/Tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,77 @@
+package BogdanSandu.tests.Tema6;
+
+import BogdanSandu.driver.BrowserManager;
+import BogdanSandu.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoAlertPresentException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+public class DemoQaWindow {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) {
+        navigateToDemoQAWindowPage();
+        try {
+            clickOnNewTabButton();
+            clickOnNewWindow();
+            FileUtils.takeScreenshot(driver, "alertDemoQa");
```

Comment:
Remove this line too as it's not needed. Screenshot should be taken only when getting an Exception that will cause test case to fail. Thanks!